File: app/src/main/java/com/ikue/japanesedictionary/fragments/SettingsFragment.java
package com.ikue.japanesedictionary.fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import com.ikue.japanesedictionary.R;

public class SettingsFragment extends PreferenceFragment {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Load the preferences from an XML resource
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: app/src/main/java/com/ikue/japanesedictionary/fragments/SettingsFragment.kt
package com.ikue.japanesedictionary.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.ikue.japanesedictionary.R

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences, rootKey)
    }
}